lib/buttons-and-links/KRouterLink.vue

```diff
@@ -6,6 +6,7 @@
     :to="to"
     :replace="replace"
     dir="auto"
+    style="display: block;"
```

Comment:
Links can't be displayed as block elements, since we have too many places where they need to be displayed as inline elements:

![image](https://user-images.githubusercontent.com/10248067/112181436-aea21300-8bb9-11eb-9cc2-8f8a1a958099.png)

Reply:
Ah. Yes, thank you! Removed.